docs/api/errors.rst

```diff
@@ -152,7 +152,7 @@ Predefined Errors
 .. autoclass:: falcon.HTTPPreconditionFailed
     :members:
 
-.. autoclass:: falcon.HTTPPayloadTooLarge
```

Comment:
Should we add a deprecated alias if anyone is still using the old name?

Reply:
Code removed